docs/topics/modules/reviewing_publishing_changes.adoc

```diff
@@ -38,7 +36,7 @@ image::expand_version.png[Expand the application version]
 . At the *Approve* stage, click btn:[Input Required].
 . Click btn:[Promote] to promote version 1.0.2 of the application to *Run*.
 +
-Your changes are now available on both *Stage* and *Run*. If you return to the *Deployments* tab, you can see the version for both *Stage* and *Run* are now 1.0.2.
+Your changes are now available on both *Stage* and *Run*. If you return to the *Deployments* tab, you can see the version for both *Stage* and *Run* are now version 1.0.2 of the application.
```

Comment:
Suggest: If you return to the *Deployments* tab, you can see version 1.0.2 of the application deployed to *Run*.

Reply:
Changed slightly to accommodate feedback